lib/CMakeLists.txt

```diff
@@ -92,6 +92,11 @@ add_library(
   ${HHEADERS} ${CSOURCES}
   )
 
+add_library(
+  CURL::${LIB_NAME}
+  ALIAS ${LIB_NAME}
+  )
```

Comment:
The `install(EXPORT)` call in the top-level `CMakeLists.txt` file uses `NAMESPACE "${PROJECT_NAME}::"`, so `CURL::` here should be written `${PROJECT_NAME}::` instead.  Similarly in `src/CMakeLists.txt`.

Actually the `export()` calls are using plain `CURL::`.  All of these places should be updated to consistently use either `CURL::` or `${PROJECT_NAME}::`.  I don't have a strong preference for which one.

Reply:
I have changed all the instances of "CURL::" to "${PROJECT_NAME}::"